src/Role/ValueObjects/Permission.php

```diff
@@ -21,6 +21,7 @@ public static function getAllowedValues(): array
             'Voorzieningen bewerken',
             'Producties aanmaken',
             'Films aanmaken',
+            'Bijdragers beheren',
```

Comment:
I was the one who suggested this extra permission, but we could also use the existing "Aanbod beheren" / "Organisaties beheren" permissions for this 🤔 The difference would of course be that with a new permission we could have more fine-grained roles, but with the latter we have less configuration work to enable this permission on existing roles. Ideally we should check this with Corneel. Maybe use the existing one for now, and add this one later in a follow-up PR if actually needed?